Makefile

```diff
@@ -1,10 +1,37 @@
 .PHONY: build_macos
 build_macos:
 	@echo "Building Macos..."
-	gogio -icon=./assets/appicon.png -target=macos -arch=amd64 -o ./dist/amd64/Chapar.app .
-	gogio -icon=./assets/appicon.png -target=macos -arch=arm64 -o ./dist/arm64/Chapar.app .
+	gogio -icon=./build/appicon.png -target=macos -arch=amd64 -o ./dist/amd64/Chapar.app .
+	gogio -icon=./build/appicon.png -target=macos -arch=arm64 -o ./dist/arm64/Chapar.app .
 	codesign --force --deep --sign - ./dist/amd64/Chapar.app
 	codesign --force --deep --sign - ./dist/arm64/Chapar.app
+	tar -cJf ./dist/Chapar_macos_amd64.tar.xz --directory=./dist/amd64 Chapar.app
+	tar -cJf ./dist/Chapar_macos_arm64.tar.xz --directory=./dist/arm64 Chapar.app
+	rm -rf ./dist/amd64
+	rm -rf ./dist/arm64
+
+.PHONY: build_windows
+build_windows:
+	@echo "Building Windows..."
+	gogio -icon=build\appicon.png -buildmode=archive -target=windows -arch=amd64 -o dist\amd64\Chapar.exe .
+	gogio -icon=build\appicon.png -buildmode=archive -target=windows -arch=386 -o dist\i386\Chapar.exe .
+	rm *.syso
+	zip -r -j dist\Chapar_windows_amd64.zip dist\amd64\Chapar.exe
+	zip -r -j dist\Chapar_windows_i386.zip dist\i386\Chapar.exe
+	rm -rf dist\amd64
+	rm -rf dist\i386
+
+.PHONY: build_linux
```

Comment:
I'm surprised by the amount code of you are adding to do build the binary?. It might be normal, but I raise the point.

I'm on phone and unable to fetch code to test for now.

Isn't the role of goreleaser to handle such things and use cases?

Is it because you have to use your docker instance that you cannot use goreleaser?

Reply:
Well goreleaser don’t have option for running custom command at least is not free

Reply:
Ah… OK… I have never faced such a need. Indeed